crates/sui-framework/sources/governance/validator.move

```diff
@@ -568,6 +568,42 @@ module sui::validator {
             // || self.metadata.net_address == other.metadata.net_address
             // || self.metadata.p2p_address == other.metadata.p2p_address
             || self.metadata.protocol_pubkey_bytes == other.metadata.protocol_pubkey_bytes
+            || self.metadata.network_pubkey_bytes == other.metadata.network_pubkey_bytes
+            || self.metadata.worker_pubkey_bytes == other.metadata.worker_pubkey_bytes
```

Comment:
Can we also compare that worker pub key and network pubkey are different?

Reply:
yes, I check that in `sui_system_state_inner_v1.rs` (since that code checks the validity of a single validator)